packages/plugin/webpack/src/WebpackPlugin.ts

```diff
@@ -30,10 +30,15 @@ export default class WebpackPlugin extends PluginBase<WebpackPluginConfig> {
   private watchers: webpack.Compiler.Watching[] = [];
   private servers: http.Server[] = [];
   private loggers: Logger[] = [];
+  private port = DEFAULT_PORT;
 
   constructor(c: WebpackPluginConfig) {
     super(c);
 
+    if (c.port) {
```

Comment:
Now I'm just being pedantic, but technically `0` is a valid port number (node will choose a random unassigned port)

Reply:
For pedantry's sake, how about it throws an error if `c.port < 1024 || c.port > 65535`? :smile:

Reply:
> For pedantry's sake, how about it throws an error if `c.port < 1024 || c.port > 65535`? :smile:

Implemented